### PR TITLE
restored --database argument in makeappointments

### PR DESCRIPTION
**Pull request by adibaejaz**

### CODE REVIEW

File: regolith/helpers/makeappointmentshelper.py [hunk 1]
```diff
@@ -33,7 +33,10 @@
 
 
 def subparser(subpi):
-
+    # Do not delete --database arg
+    subpi.add_argument("--database",
```
Comment (sbillinge) [inline regolith/helpers/makeappointmentshelper.py:37]: please put this at the bottom of the list so it appears last in the help.  It is rarely used (by design)

File: regolith/helpers/makeappointmentshelper.py [hunk 2]
```diff
@@ -33,7 +33,10 @@
...
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file.")
     subpi.add_argument("run", help='run the helper. to see optional arguments, enter "regolith helper makeappointments"')
     subpi.add_argument("--no_plot", action="store_true", help='suppress plotting feature')
     subpi.add_argument("--no_gui", action="store_true", help='suppress interactive matplotlib GUI')
```
Comment (sbillinge) [inline regolith/helpers/makeappointmentshelper.py:42]: please add "(used for running tests)" to the help on this.

File: regolith/helpers/makeappointmentshelper.py (hunk 1 above)
Comment (adibaejaz) [reply regolith/helpers/makeappointmentshelper.py:41]: done

File: regolith/helpers/makeappointmentshelper.py (hunk 2 above)
Comment (adibaejaz) [reply regolith/helpers/makeappointmentshelper.py]: done

**Comment by sbillinge** [comment]
thanks @adibaejaz

**Pull request merged**